mne/io/write.py

```diff
@@ -194,6 +194,26 @@ def write_int_matrix(fid, kind, mat):
     check_fiff_length(fid)
 
 
+def write_complex_matrix(fid, kind, mat):
+    """Write complex 64 matrix tag."""
+    FIFFT_MATRIX = 1 << 30
+    FIFFT_MATRIX_FLOAT = FIFF.FIFFT_FLOAT | FIFFT_MATRIX
```

Comment:
there is constants for complex numbers:

FIFF.FIFFT_COMPLEX_FLOAT         = 20
FIFF.FIFFT_COMPLEX_DOUBLE        = 21

look in constants.py